staging/src/k8s.io/kube-aggregator/pkg/apiserver/handler_apis.go

```diff
@@ -42,25 +43,39 @@ type apisHandler struct {
 	discoveryGroup metav1.APIGroup
 }
 
-func discoveryGroup(enabledVersions sets.String) metav1.APIGroup {
+// discoveryGroup generates the APIGroup for the apiregistration group
+// (i.e. the API Registration API, which is responsble for registering an API
+// `Group`/`Version` with other kubernetes like API servers). It is called
+// when instantiating the APIAggregator.
+// enabledVersions is used to determine if the v1beta1 version of the apiregistration
+// group is also the be included the resulting APIGroup.
+// handler is a GenericAPIServerHandler, used to resolve the hash of the API GroupVersion
+func discoveryGroup(enabledVersions sets.String, handler http.Handler) metav1.APIGroup {
+	hash, err := discovery.GetGroupVersionHash("/apis/"+apiregistrationv1api.SchemeGroupVersion.String(), handler)
+	if err != nil {
+		panic(err)
```

Comment:
If there is an error grabbing the hash I'm not sure the error should be treated with a panic. Given that a missing hash is not a correctness issue, but a performance one; I would be inclined to log it as an error and allow the group to exist without a hash.